src/index.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+
+test('renders learn react link', () => {
+  const { getByText } = render(
+    <Provider store={store} />,
+  );
+
+  expect(getByText(/learn/i)).toBeInTheDocument();
+});
```

Comment:
- [ ] Please this test is failing, kindly  fix the issue. See image below;
![test](https://user-images.githubusercontent.com/65852621/147077740-71d9fe39-576f-4ed3-a6dd-944a600f944c.png)

- [ ] As it is a requirement for this project, please kindly add more tests to increase your test coverage, this will reduce the chances of your app breaking in production. See screenshots of the requirements below. 

![testREQ](https://user-images.githubusercontent.com/65852621/147078358-515b0b76-7d58-4367-814f-98947c695311.png)

![testREQ2](https://user-images.githubusercontent.com/65852621/147078383-b4eefef7-93f7-4feb-9fd5-9e48cb09a7d4.png)